src/simplewallet/simplewallet.cpp

```diff
@@ -3751,93 +3755,46 @@ bool simple_wallet::init(const boost::program_options::variables_map& vm)
 
     if (m_restoring && m_generate_from_json.empty() && m_generate_from_device.empty())
     {
-      m_wallet->explicit_refresh_from_block_height(!(command_line::is_arg_defaulted(vm, arg_restore_height) ||
```

Comment:
Note the restore date param, need to bring this back if that arg is not gonna be removed in this commit